src/Uno.UI.Runtime.Skia.Wpf/Extensions/UI/Xaml/Controls/TextBoxViewExtension.cs

```diff
@@ -160,7 +164,19 @@ public void SetIsPassword(bool isPassword)
 			// No support for now.
 		}
 
-		public void Select(int start, int length) => _currentInputWidget?.Select(start, length);
+		public void Select(int start, int length)
+		{
+			_currentInputWidget?.Select(start, length);
+
+#if __SKIA__
```

Comment:
When can this be `false`?

Reply:
When can this be false? I don't think we're reusing this file anywhere but here. You'll probably be able to remote the #if.